x/mint/simulation/genesis.go

```diff
@@ -140,11 +138,6 @@ func RandomizedGenState(simState *module.SimulationState) {
 
 	mintGenesis := types.NewGenesisState(minter, params, reductionStartedEpoch)
 
-	bz, err := json.MarshalIndent(&mintGenesis, "", " ")
-	if err != nil {
-		panic(err)
-	}
-	fmt.Printf("Selected pseudo-randomly generated minting parameters:\n%s\n", bz)
```

Comment:
Driveby deletion, to start stopping this extremely unhelpful pattern from the SDK